### DISCUSSION TITLE
Trackmania nations forever support

### DISCUSSION THREAD

**Discussion by Me123456789010** [trackmania-rl/tmrl#59] [Q&A] [answered]
Does the plugin support trackmania nations forever or other versions of trackmania like turbo...

**Comment by yannbouteiller** [comment, marked as answer]
We started this project on TMNF, very old versions of tmrl have support for it but we removed that after moving to full support for TM2020.

In particular, OpenPlanet is not compatible with TMNF, so our OpenPlanet plugin does not support it. In fact there are tools for TMNF that allow you to hack deeper into the game and do interesting things such as spawning the car at given positions/speeds or pausing/accelerating the game, but tmrl is more concerned about real-time robotics than game-hacking, so we did not look into that.

**Reply by Me123456789010** [reply]
Also, do you know if Trackmania Club is required for the plugin? I am using the free version and no plugin is visible
<img width="380" alt="image" src="https://github.com/trackmania-rl/tmrl/assets/145355761/2e436a39-9ab8-4eed-9bf9-ff194575a181">
P.S.=> the signature mode is regular.

**Reply by yannbouteiller** [reply]
The plugin should work with the free version, as it is signed by OpenPlanet.

Maybe double-check that it is here in the `OpenPlanetNext/Plugins` folder?

If it is there, you should see it appear in `Load plugin` or `Toggle plugin`

**Reply by Me123456789010** [reply]
No its not there. Is it available on github??

**Reply by yannbouteiller** [reply]
Yes but you should have a copy in your `TmrlData` folder under `resources`

**Reply by Me123456789010** [reply]
Yes hhankyYou very much for your help
